.github/workflows/unit.yml

```
@@ -1,14 +1,10 @@
 name: Unit Tests
 
 on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+ workflow_call:
+   git
```

actionlint (1.7.12)

[error] "workflow_call" section is scalar node but mapping node is expected

(syntax-check)